src/pages/Profile/Profile.css

```diff
@@ -0,0 +1,24 @@
+.profile-pic {
+  position: relative;
+  top: 70px;
+  left: 500px;
+}
+
+.user {
+  position: absolute;
+  right: 700px;
+  top: 70px;
+}
+
+.track-progress {
+  position: relative;
+  left: 300px;
+  top: 85px;
+}
+
+.progressBar {
+  width: 30%;
+  position: relative;
+  left: 300px;
+  top: 90px;
+}
```

Comment:
🔧 Let's use Material-UI built-in layout tools instead of custom css.
https://material-ui.com/components/box/

Reply:
I would LOVE to do that, could you show me how, please?